android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/components/EmojiRatingAnswer.kt

```diff
@@ -83,7 +103,39 @@ fun LeftEmojiRatingAnswerPreview(
         emojis = List(5) {
             { Emoji(name = emojiName) }
         },
-        onIndexChange = {}
+        answers = listOf(
+            Answer(
+                id = "1",
+                text = "choice 1",
+                displayOrder = 0,
+                inputMaskPlaceholder = null
+            ),
+            Answer(
+                id = "2",
+                text = "Choice 2",
+                displayOrder = 1,
+                inputMaskPlaceholder = null
+            ),
+            Answer(
+                id = "3",
+                text = "choice 3",
+                displayOrder = 0,
+                inputMaskPlaceholder = null
+            ),
+            Answer(
+                id = "4",
+                text = "Choice 4",
+                displayOrder = 1,
+                inputMaskPlaceholder = null
+            ),
+            Answer(
+                id = "5",
+                text = "choice 5",
+                displayOrder = 0,
+                inputMaskPlaceholder = null
+            )
+        ),
```

Comment:
optional: in case you want to optimize, we can build this list with a loop from 1 to 5

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/123/commits/07f6a3df329b5ed79084a3fe821669529cd47d33